.github/workflows/publish-release-artifacts.yml

```diff
@@ -0,0 +1,180 @@
+name: Publish Release Artifacts
+
+on:
+  workflow_dispatch:
+    inputs:
+      ballerina_version:
+        description: 'Ballerina Version'
+        default: '2201.1.1'
+        required: true
+      short_version:
+        description: 'Short Version'
+        default: '2201.1.1'
+        required: true
+      display_version:
```

Comment:
@NipunaMadhushan Currently we only show version in dist list. So we should be able to remove this